libraries/timeline-view/src/main/java/com/trendyol/timelineview/TimelineItemViewState.kt

```diff
@@ -22,15 +21,15 @@ class TimelineItemViewState(
 
     fun getTextColor(): Int = Color.parseColor(timelineItem.textColor)
 
-    fun getLeftLineColor(): Int = Color.parseColor(timelineItem.leftLineColor ?: hexColorWhite)
+    fun getLeftLineColor(): Int = Color.parseColor(timelineItem.startLineColor ?: hexColorWhite)
```

Comment:
Your function not responsible for only left/right. Please find more declarative function naming.

Reply:
wrong function names fixed